src/Microsoft.TestPlatform.CoreUtilities/Helpers/Interfaces/IDotnetHostHelper.cs

```diff
@@ -27,7 +28,11 @@ public interface IDotnetHostHelper
     /// Try to locate muxer of specific architecture
     /// </summary>
     /// <param name="targetArchitecture">Specific architecture</param>
+    /// <param name="dotnetMuxerResolutionStrategy">The dotnet muxer resolution strategy.</param>
     /// <param name="muxerPath">Path to the muxer</param>
     /// <returns>True if native muxer is found</returns>
-    bool TryGetDotnetPathByArchitecture(PlatformArchitecture targetArchitecture, out string? muxerPath);
+    bool TryGetDotnetPathByArchitecture(
+        PlatformArchitecture targetArchitecture,
+        DotnetMuxerResolutionStrategy dotnetMuxerResolutionStrategy,
```

Comment:
Can we add the default to `Default`?

Reply:
We cannot do this since that would make the `DotnetMuxerResolutionStrategy` parameter optional, but  since it is not the last argument, the build will fail.

Reply:
ah yeah you're right